package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "threestrap",
-  "version": "0.4.1",
+  "version": "0.5.0",
```

Comment:
I went with a minor version bump since this is breaking, I guess. (But still in the 0 range.)